src/deepsparse/pipeline.py

```diff
@@ -175,6 +176,8 @@ def __init__(
 
         self.onnx_file_path = self.setup_onnx_file_path()
         self.engine = self._initialize_engine()
+        self._inference_timing = None
```

Comment:
inference timing should not be a class property since it relates to a single inference - should be returned with the specific inference call